prediction_market_agent/agents/known_outcome_agent/deploy.py

```diff
@@ -17,7 +17,8 @@
 
 
 class DeployableKnownOutcomeAgent(DeployableAgent):
-    model = "gpt-4-turbo-preview"
+    model = "gpt-4-1106-preview"
```

Comment:
Needed to change this, see the `summarize_if_required` function below. Lmk if I did wrong 🙈

Reply:
oops, my bad!